pandora-client-web/src/components/wardrobe/views/wardrobeLockLogic.tsx

```diff
@@ -200,13 +210,35 @@ export function WardrobeLockLogicUnlocked<TActionContext>({ lockLogic, ActionBut
 			setUseOldPassword(false);
 	}, [lockLogic.hasPassword]);
 
+	const timerOptions = useMemo((): LockTimerOptions => {
+		return {
+			timer,
+			allowEarlyUnlock: timerAllowEarly,
+		};
+	}, [timer, timerAllowEarly]);
+
+	const setTimerAllowEarlyWithConfirm = useCallback((newValue: boolean) => {
+		if (newValue) {
+			setTimerAllowEarly(newValue);
+			return;
+		}
+
+		confirm('Confirm disallow early unlock', <>Are you sure you want to disallow the locker to unlock before the timer runs out?</>)
```

Comment:
Similarly updating the text here:
```suggestion
		confirm('Confirm disallowing early unlock', <>Are you sure you want to prevent yourself from being able to unlock the lock before the timer runs out?</>)
```